### PR TITLE
Add `--subdomain` option for genezio deploy

### PR DESCRIPTION
**Pull request by costinsin**
## Type of change

-   [x] 🍕 New feature

## Description
This change adds a new `--subdomain` option flag. This option overrides the `genezio.yaml` configuration file subdomain entry.

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Comment by costinsin** [comment]
Current dependencies on/for this PR:
* dev
  * **PR #556** <a href="https://app.graphite.dev/github/pr/Genez-io/genezio/556?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/Genez-io/genezio/556?utm_source=stack-comment).

File: src/commands/deploy.ts [hunk 1]
```diff
@@ -573,14 +573,14 @@ export async function deployFrontend(
             log.info("WARNING: No index.html file found in the build folder");
         }
 
+        configuration.frontend.subdomain = options.subdomain || configuration.frontend.subdomain;
         if (!configuration.frontend.subdomain) {
             log.info(
-                "No subdomain specified in the genezio.yaml configuration file. We will provide a random one for you.",
+                "No subdomain specified in the genezio.yaml configuration file or as a option flag. We will provide a random one for you.",
```
Comment (andreia-oca) [inline src/commands/deploy.ts:579]:
```suggestion
                "No subdomain specified in the genezio.yaml configuration file or as an option flag. We will provide a random one for you.",
```

**Review by andreia-oca — APPROVED** [review]
[beautiful](https://app.graphite.dev/macros/Genez-io)

**Comment by costinsin** [comment]
### Merge activity

* **Nov 21, 6:42 AM**: @costinsin merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Genez-io/genezio/556).

**Pull request merged**